invite/test/e2e.test.ts

```diff
@@ -91,7 +100,7 @@ describe('end-to-end', () => {
           .reply(200);
 
         await triggerWebhook(probot, 'trigger_invite.issue_comment.created');
-        expect(await db('invites').select().first()).toBeUndefined();
+        expect(await record.getInvites('someone')).toEqual([]);
```

Comment:
Using the `InvitationRecord` interface here simplifies the checks, since that class handles some typecasting nonsense from Postgres

Reply:
Nit: I prefer using `expect(foo).resolves.bar` instead of `expect(await foo).bar`:
```typescript
expect(record.getInvites('someone')).resolves.toBe([]);
```

(yes, I am aware my old tests aren't using this. I learned about this after writing them 🙃)

Reply:
> Using the `InvitationRecord` interface here simplifies the checks, since that class handles some typecasting nonsense from Postgres

+1

Reply:
Modified all tests (in `invite_bot` and elsewhere) to use that form.